examples/select/option-symbols.js

```diff
@@ -18,21 +16,12 @@ const prompt = new Prompt({
     muted: colors.yellow,
   },
   symbols: {
-    radio: {
-      on: state => ['🍬', '🍎', '👄', '🖕'][state.index],
-      off: '  '
-    }
-  },
-  elements: {
-    prefix: (state, status) => emoji[status],
-    pointer(state, status, choice) {
-      let symbol = this.symbols.radio[status];
-      let fallback = status === 'on' ? '🗡️ ' : '  ';
-      if (typeof symbol === 'function') {
-        return symbol(...arguments) || fallback;
-      }
-      return symbol || fallback;
+    pointer(state) {
+      return ['🍬', '🍎', '👄', '🖕', '🗡️ '][state.index];
     },
+    prefix(state) {
+      return ({ pending: '🎃', cancelled: '⚰️ ', answered: '💀' })[state.status];
```

Comment:
This is probably one of my favorite examples 🎃

Reply:
Yeah that one is fun :)